sdk/appcenter-analytics/src/test/java/com/microsoft/appcenter/analytics/AnalyticsTransmissionTargetTest.java

```diff
@@ -436,4 +442,87 @@ public void eventPropertiesCascading() {
         expectedProperties.put("g", "7777");
         assertEquals(expectedProperties, log.getProperties());
     }
+
+    /* Property Configurator tests */
+    @Test
+    public void setCommonSchemaProperties() {
+        CommonSchemaLog log = new CommonSchemaEventLog();
+        log.setExt(new Extensions());
+        log.getExt().setApp(new AppExtension());
+
+        /* Get property configurator and set properties. */
+        PropertyConfigurator pc = Analytics.getTransmissionTarget("test").getPropertyConfigurator();
+        pc.setAppVersion("appVersion");
+        pc.setAppName("appName");
+        pc.setAppLocale("appLocale");
+        pc.onPreparingLog(log, "groupName");
+
+        /* Assert properties set on common schema. */
+        assertEquals("appVersion", log.getExt().getApp().getVer());
+        assertEquals("appName", log.getExt().getApp().getId());
+        assertEquals("appLocale", log.getExt().getApp().getLocale());
+    }
+
+    @Test
+    public void inheritCommonSchemaProperties() {
```

Comment:
This test should be removed because it's redundant with `inheritCommonSchemaPropertiesFromGrandparent`.  It misses a small bit of code coverage which is covered by the other test